README.rst

```diff
@@ -795,7 +793,7 @@ Linux with vlan interface_params
           vlan69:
             type: vlan
             use_interfaces:
-            - interface: ${linux:interface:bond0}
+            - bond0
```

Comment:
this change must be tested how it affects current current deployments

Reply:
I doubt current deployments are working, if they were configured like the example in the readme